tfjs-converter/src/executor/graph_executor.ts

```diff
@@ -256,6 +276,10 @@ export class GraphExecutor implements FunctionExecutor {
       if (this.parent == null) {
         context.dispose(tensorsToKeep);
       }
+      if (this.keepIntermediateTensors) {
+        this.clonedTensorsMap = this.cloneTensorMap(tensorsMap);
+      }
```

Comment:
nit: Should this `if` happen before L276?

Reply:
done